relayer/src/event/rpc.rs

```diff
@@ -11,6 +11,106 @@ use ibc::events::{IbcEvent, RawObject};
 
 use crate::event::monitor::queries;
 
+/// Extract IBC events from Tendermint RPC events
+///
+/// Events originate from the following ABCI methods ->
+/// 1. `DeliverTx` - these events are generated during the execution of transaction messages.
+/// 2. `BeginBlock`
+/// 3. `EndBlock`
+///
+/// Events originating from `DeliverTx` are currently extracted via the `RpcEvent::data` field as
+/// the `EventData::Tx` variant.
+/// Here's an example of what these events look like (i.e. `TxInfo::TxResult::events`) -
+/// ```ron
```

Comment:
nit: json instead of ron?

Reply:
Sorry for not addressing this - the snippet is just debug output, so it's closer to `ron` than it is to `json`.

Reply:
No worries, it's fine as is imho